.tool-versions

```diff
@@ -1,2 +1,2 @@
-elixir 1.11.0-otp-23
-erlang 23.1.1
+elixir 1.12.1-otp-24
```

Comment:
Don't we have Elixir 1.11 in production? I think is better to use the prd version

Reply:
PEA will use 1.12 in production once we migrate to Kubernetes.

Reply:
Is it ready to merge then? @Tolsto